meilisearch-http/src/routes/stats.rs

```diff
@@ -115,20 +125,18 @@ struct VersionResponse {
     pkg_version: String,
 }
 
-pub async fn get_version(ctx: Request<Data>) -> SResult<Response> {
-    ctx.is_allowed(Admin)?;
-    let response = VersionResponse {
+#[get("/version", wrap = "Authentication::Private")]
```

Comment:
Is it wanted to downgrade the privacy to private instead of admin? I remember we talked about this and I think this is safe.

Reply:
Yes, it's an open issue https://github.com/meilisearch/MeiliSearch/issues/584

Reply:
I am aware of this issue, it doesn't tell if we need to change the privacy level, but if you want to change it I am not against that.

Reply:
If I change it now, we'll be up to date with the doc.